lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -3574,11 +3574,11 @@ void addPassesForSPIRV(legacy::PassManager &PassMgr,
   if (Opts.isSPIRVMemToRegEnabled())
     PassMgr.add(createPromoteMemoryToRegisterPass());
   PassMgr.add(createPreprocessMetadata());
-  PassMgr.add(createOCL21ToSPIRV());
+  //PassMgr.add(createOCL21ToSPIRV());
```

Comment:
```suggestion

```

Reply:
Thanks, fixed in 7694b73631bad0e52217cda32365d382e35bb768